Dockerfile

```diff
@@ -4,13 +4,15 @@ COPY . .
 RUN make
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+ARG CSV=4.1
 
 ENV ALERTS_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_alerts.yml"
 ENV RULES_FILE_PATH="/etc/elasticsearch-operator/files/prometheus_rules.yml"
 
 COPY --from=builder /go/src/github.com/openshift/elasticsearch-operator/_output/bin/elasticsearch-operator /usr/bin/
-COPY files/ /etc/elasticsearch-operator/files/
-ADD controller-manifests /manifests
+COPY --from=builder files/ /etc/elasticsearch-operator/files/
+COPY --from=builder manifests/elasticsearch-operator.package.yaml /manifests
```

Comment:
isn't this creating a file named "/manifests"?  hence why your next command fails because /manifests isn't a dir, it's a yaml file.

Reply:
i dont know why you wouldn't hit the same issue in https://github.com/openshift/cluster-logging-operator/pull/163/files#diff-3254677a7917c6c01f55212f86c57fbfR19

Reply:
(put a trailing / on it in both PRs to enforce it as a dir)